### DISCUSSION TITLE
Brimstone Ring instead of Berserker Ring (i) Indicator

### DISCUSSION THREAD

**Discussion by Quoze** [runelite/runelite#16104] [Ideas]
There are some occasions where wearing a berserker ring (i) instead of brimstone ring does not give any max hits so wearing the brimstone would be better for the attack and defence bonuses. 

Assuming divine supercombat potted (and maybe a toggle for cox overload and toa salts too) it would be cool that (when wearing a berserker ring) a little infobox or something small on the screen popped up to indicate with your current gear, a brimstone ring would give the same max hit

**Comment by Enriath** [comment]
This would be better solved by the user using a DPS calculation spreadsheet, or as an addition to an existing hub plugin.
